### PR TITLE
creating base docker image for cals builds

### PR DESCRIPTION
**Pull request by ratneshraval**
#### What tests were added (if relevant), and do all tests (rspec, karma, and integration CALS tests) pass?
- Hope so!

#### Which User story does this relate to?
- https://www.pivotaltracker.com/story/show/151825351

#### Brief feature description
- Create a base image so each build takes less time

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,47 +1,4 @@
...
-RUN wget 'https://drive.google.com/uc?export=download&id=0B8styZ_fXfKEQkFPTldZWkRYV3M' -O /usr/local/share/$PHANTOM_JS.tar.bz2
-
-WORKDIR /usr/local/share/
-
-RUN cd /usr/local/share/ \
-  && tar xvjf $PHANTOM_JS.tar.bz2 \
-  && ln -sf /usr/local/share/$PHANTOM_JS/bin/phantomjs /usr/local/share/phantomjs \
-  && ln -sf /usr/local/share/$PHANTOM_JS/bin/phantomjs /usr/local/bin/phantomjs \
-  && ln -sf /usr/local/share/$PHANTOM_JS/bin/phantomjs /usr/bin/phantomjs
+FROM ratneshraval/cals-base:0.2
```
Comment (lavanyareddyreddy) [inline Dockerfile]: Is this base image name?

Comment (ratneshraval) [reply Dockerfile]: Yes, once approved it will change to cwds/cals-base:0.2

Comment (ratneshraval) [reply Dockerfile]: once approved by dev-ops/security.

Comment (lavanyareddyreddy) [reply Dockerfile]: Ok.

**Pull request merged**